packages/studio/frontend/package.json

```
@@ -7,12 +7,19 @@
     "watch": "yarn && yarn run -T parcel watch src/index.html",
     "dev": "yarn && yarn run -T parcel src/index.html",
     "build:scss-types": "typed-scss-modules src --logLevel error --nameFormat none --exportType default",
-    "build:tsc": "yarn run -T tsc --build"
+    "build:tsc": "yarn run -T tsc --build",
+    "storybook": "yarn run -T start-storybook -p 6006 -c story/config",
+    "build-storybook": "yarn run -T build-storybook -c story/config"
```

What do you think of having those commands instead?

```suggestion
    "start:storybook": "yarn run -T start-storybook -p 6006 -c story/config",
    "build:storybook": "yarn run -T build-storybook -c story/config"
```